qlib/rl/order_execution/state.py

```diff
@@ -38,6 +39,18 @@ def _get_all_timestamps(
     return pd.DatetimeIndex(ret)
 
 
+def fill_missing_data(
```

Comment:
I think making exchange return data without missing will make it more elegant.
Please add TODO here.